src/logger.js

```diff
@@ -0,0 +1,7 @@
+import winston from 'winston';
+
+export const logger = winston.createLogger({
+    transports: [
```

Comment:
you can also modify your logging format by providing some essential data like timestamp
```
format: winston.format.combine(
    winston.format.timestamp(),
    winston.format.json()
)
```

Reply:
Added this, thanks for sharing!